### DISCUSSION TITLE
[Beta 2.0.0] Add asset/matrix field without removing previous content

### DISCUSSION THREAD

**Discussion by RichardGG** [engram-design/FeedMe#871] [Ideas]
Is it possible to update an entry's asset field without removing the previous content?

Here is an example:
I have a set of entries with a unique gallery ID and an asset field

Using my JSON feed, I can update each gallery entry, but the image replaces the previous asset rather than simply adding it to the field.

`[ {"gallery_id": 1, "image": "url1.jpg"}, {"gallery_id": 1, "image": "url2.jpg"} ]`

Perhaps this could be an alternative import strategy?

**Comment by engram-design** [comment]
Its not currently possible to append content on element fields (Assets, Entries, Matrix, etc). Data will instead entirely replace the content of these fields. I'll look into adding this as an option for each element field, as its not really about the import strategy as a whole.

**Comment by huelabs** [comment]
Sometimes us content creators need to append a value to an existing set of data. This would be super useful.

**Comment by tibbis** [comment]
Poking up this again, my use case is to append array data to a matrix field array on multiple feed runs. 

But I would add a unique identifier to each item to check each block so you can update items still, exactly how you do it for the base entry in FeedMe.
